docs/decisions/0011_multiple_event_types_per_topic.rst

```diff
@@ -0,0 +1,73 @@
+10. Multiple event types per topic
+##################################
+
+Status
+******
+
+- **Accepted** DATE TO BE ADDED BEFORE MERGING
+
+Context
+*******
+
+Following `this ADR`_ in openedx-events, all implementations of event bus must support multiple event types on the same topic. This ADR explains how we will implement this using Kafka.
+**UPDATE LINK AFTER THAT ADR MERGES**
+
```

Comment:
Can you add a small section providing a brief list of possible solutions to this problem in Kafka? Could just be bullets, and you can note that they'll be covered in more detail later. But basically, the context is that there are 2 (or 3, etc. depending on how you look at it) ways of accomplishing this, and this ADR is just about the one we selected.

Reply:
[fixed]